client/my-sites/plans-v2/constants.ts

```diff
@@ -301,19 +301,23 @@ export const EXTERNAL_PRODUCT_CRM: SelectorProduct = {
 		? 'jetpack_crm_dark'
 		: 'jetpack_crm',
 	displayName:
-		getJetpackCROActiveVersion() === 'v2'
-			? translate( 'Jetpack CRM {{em}}Entrepreneur{{/em}}', {
-					components: {
-						em: createElement( 'em' ),
-					},
-			  } )
-			: translate( 'Jetpack CRM' ),
+		{
+			v2: translate( 'Jetpack CRM {{em}}Entrepreneur{{/em}}', {
+				components: {
+					em: createElement( 'em' ),
+				},
+			} ),
+			i5: translate( 'CRM Entrepreneur' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'CRM: Entrepreneur bundle' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/improve/pricing-page-i5-update-copy)